meerdag/block.go

```diff
@@ -23,6 +23,8 @@ type IBlockData interface {
 
 	// Priority
 	GetPriority() int
+
+	Clean()
```

Comment:
add `Clean()` method into this interface might be add the complex to maintain in the future.  we should not mix the business abstract interface with the implement issue (like the memory garbage collect and cleanup)

Reply:
👌